projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -219,9 +220,10 @@ if (environment.s4om) {
     ImageZoomFeatureModule,
 
     CustomerTicketingFeatureModule,
+
     ProductConfiguratorTextfieldFeatureModule,
     ProductConfiguratorRulebasedFeatureModule,
-
+    PickupInStoreFeatureModule,
```

Comment:
if `PickupInStoreFeatureModule` is always imported, why do we still need `pickupInStore` in `environment`?

Reply:
We can address this after the merge, either we remove the flag and leave it always there or we figure out how to conditionally add with tests passing in the pipeline.